src/sciline/pipeline.py

```diff
@@ -710,6 +725,8 @@ def get(
             During development and debugging it can be helpful to use a handler that
             raises an exception only when the graph is computed. This can be achieved
             by passing :py:class:`HandleAsComputeTimeException` as the handler.
+        check:
+            Checks if the constructed graph uses all available parameters.
```

Comment:
In #43, detecting missing providers was also mentioned, so maybe we need a more specific name for unused parameters check here.

Reply:
Yes, we need a more meaningful name.